### ISSUE TITLE
Subworkflow without a configfile gives error

### ISSUE THREAD

**Issue by SultanOrazbayev** [snakemake/snakemake#24]
**Snakemake version**
5.7.0

**Describe the bug**
If a subworkflow does not have a `configfile` entry, an error is shown.

**Logs**

```
Traceback (most recent call last):
  File "/mypath/miniconda/envs/mydef/lib/python3.7/site-packages/snakemake/__init__.py", line 611, in snakemake
    export_cwl=export_cwl,
  File "/mypath/miniconda/envs/mydef/lib/python3.7/site-packages/snakemake/workflow.py", line 558, in execute
    updated_files=updated,
  File "/mypath/miniconda/envs/mydef/lib/python3.7/site-packages/snakemake/__init__.py", line 389, in snakemake
    overwrite_config.update(load_configfile(f))
  File "/mypath/miniconda/envs/mydef/lib/python3.7/site-packages/snakemake/io.py", line 1303, in load_configfile
    config = _load_configfile(configpath)
  File "/mypath/miniconda/envs/mydef/lib/python3.7/site-packages/snakemake/io.py", line 1265, in _load_configfile
    with open(configpath) as f:
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

**Minimal example**

Running this code gives error (if `configfile` entry is copied to the subworkflow, then the error will disappear):
```python
configfile: "config.yml"

subworkflow test:
    snakefile: "sub.snakefile"

rule all:
    input: test('test.txt')
```

The `sub.snakefile` can be:
```python
rule all:
    output: touch('test.txt')
```

**Comment by wclose** [comment]
Plus one to this bug. Just found it myself.

**Comment by wclose** [comment]
@johanneskoester does this have to do with the brackets on line 561 of workflow.py? 
```
configfiles=[subworkflow.configfile],
```
Based on the blame history, it looks like it's something that was changed in PR #3

**Comment by SultanOrazbayev** [comment]
@wclose: thanks! removing the brackets resolves the issue. I'm running the tests to see if that introduces problems anywhere else, and if not will submit PR.

**Comment by VukanJ** [comment]
In 5.10.0 this is still an issue

**Comment by johanneskoester** [comment]
Fixed with PR #271.
